src/DynamoManipulation/Gizmo.cs

```diff
@@ -163,7 +163,7 @@ internal Gizmo(NodeManipulator manipulator)
 
         private void Redraw()
         {
-            if (manipulator.Enabled)
+            if (manipulator.IsEnabled())
```

Comment:
so this happens whenever the camera is moved?

Reply:
This call takes place for every camera changed event and there's another call to `IsEnabled()` in `MouseMove()`, however since the `isValidNode` field is cached, these methods will not query for the node's cached value each time.